hack/ci-e2e.sh

```diff
@@ -11,6 +11,9 @@
 set -eux
 
 REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/..")
+BMO_IMAGE_NAME="quay.io/metal3-io/baremetal-operator"
```

Comment:
Ah, yes. Initially I intended to use this var for the UPGRADE_FROM image as well, but turned out we don't need to fetch it beforehands, minikube can do that automatically.

Reply:
Yes that is expected since we upgrade from an existing release that already has a public image. The reason we need the `e2e` image normally is because we must run the tests with the very latest code from the current PR :slightly_smiling_face: